README.md

```diff
@@ -1,6 +1,6 @@
 # Shaderc
 
-[![Build Status](https://travis-ci.org/google/shaderc.svg)](https://travis-ci.org/google/shaderc)
+[![Build Status](https://travis-ci.org/google/shaderc.svg)](https://travis-ci.org/google/shaderc) [![Build status](https://ci.appveyor.com/api/projects/status/xcxci2mekggib1an?svg=true)](https://ci.appveyor.com/project/antiagainst/shaderc)
```

Comment:
They should probably have different text, to suggest the difference between the two.  Maybe "Linux build status" and "Windows build status"?